src/main/menu.js

```diff
@@ -221,13 +219,20 @@ export const setupMenu = store => {
     );
   }
 
-  // Check for Updates...
-  defaultTemplate[isOSX() ? 0 : 4].submenu.splice(isOSX() ? 1 : 0, 0, {
-    label: label('app.check-for-updates'),
-    click: () => {
-      checkForUpdates();
-    }
-  });
+  // About and Check for Updates...
+  // App menu on macOS and Help menu on others
+  defaultTemplate[isOSX() ? 0 : 4].submenu.splice(
```

Comment:
Why splice? It _looks_ like you're either duplicating the array or inserting at the beginning. Either way, why not use `unshift(item1, item2...)`?